src/Orb/Handler/PermissionAction.hs

```diff
@@ -32,10 +35,21 @@ import Orb.Handler.PermissionError (PermissionError)
 -}
 class PermissionError (PermissionActionError action) => PermissionAction action where
   -- |
-  --     'PermissionActionMonad' is an associated type that indicates the monad in which
+  --     'permissionactionmonad' is an associated type that indicates the monad in which
   --     the permission check will be executed.
   type PermissionActionMonad action :: Type -> Type
 
+  -- |
+  --     'PermissionActionMonad' is an associated type that indicates the monad in which
+  --     the handle executes (if it's different than the permission action monad). Because
```

Comment:
```suggestion
  --     'PermissionActionHandlerMonad' is an associated type that indicates the monad in which
  --     the handler executes (if it's different than the permission action monad). Because
```